python/python/glide/async_commands/core.py

```diff
@@ -71,6 +71,23 @@ class ExpiryType(Enum):
     KEEP_TTL = 4, Type[None]  # Equivalent to `KEEPTTL` in the Redis API
 
 
+class ExpiryTypeGetEx(Enum):
+    """GetEx option: The type of the expiry.
```

Comment:
Please rename to `ExpiryType` - could be used in another commands too (restore, set)

Reply:
Set already has an `ExpiryType`, just slightly different

Reply:
It has an element thats not an option in `GetEx`, and vice versa the `PERSIST` option isnt available for `Set`

Reply:
OMFG
If we move `PERSIST` and `KEEPTTL` from the enum, we can reuse it.
Can you clarify possibility of that? If we can't need to confirm naming

Reply:
Redis API is THE BEST

Reply:
> Can you clarify possibility of that?

I believe that's doable, but it'll obviously have to come with some refactoring of `SET`. Like andrew said we could keep the shared part in the enum and make KEEPTTL/PERSIST their own thing, making a time value not mandatory when choosing these options.

Reply:
Will leave the refactoring to a future task to keep consistency here. tracked in [this internal ticket.](https://github.com/orgs/Bit-Quill/projects/5/views/1?filterQuery=-status%3A%22NEEDS+GROOMING%22+&pane=issue&itemId=67911212)

Reply:
Renaming later (after python client release) would cause a breaking change.
But I don't insist